### PR TITLE
 Layout draft

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Roboto', sans-serif;
+}
```
Comment (mi-chal) [inline src/index.scss]: You've got `body` duplicated. It's here and in index.scss

File: src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import App from 'App';
+
+import 'index.scss';
```
Comment (mi-chal) [inline src/index.jsx]: This won't work.

File: src/App.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import AbcCams from 'screens/AbcCams/AbcCams/AbcCams';
+
+import 'App.scss';
```
Comment (mi-chal) [inline src/App.jsx]: This willl not work

File: src/App.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import AbcCams from 'screens/AbcCams/AbcCams/AbcCams';
```
Comment (mi-chal) [inline src/App.jsx]: What's that? `AbcCams/AbcCams/AbcCams'`

File: src/App.jsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+import 'App.scss';
+import { BrowserRouter } from 'react-router-dom';
```
Comment (mi-chal) [inline src/App.jsx]: Imports order

File: src/screens/AbcCams/AbcCams/styles.scss [hunk 6]
```diff
@@ -0,0 +1,19 @@
+.main {
+  text-align: center;
+  background-color: #F8F8F8;
```
Comment (mi-chal) [inline src/screens/AbcCams/AbcCams/styles.scss]: Colors should be imported from `colors.scss`

File: src/screens/AbcCams/AbcCams/styles.scss [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  min-height: 70vh;
+}
+
+.footer {
+  display: flex;
+  background-color: #282828;
```
Comment (mi-chal) [inline src/screens/AbcCams/AbcCams/styles.scss]: Colors should be imported from `colors.scss`

File: src/screens/AbcCams/AbcCams/styles.scss [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+  color: white;
```
Comment (mi-chal) [inline src/screens/AbcCams/AbcCams/styles.scss]: Colors should be imported from `colors.scss`

File: src/shared/components/Navigation/styles.scss [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+  flex-grow: 1;
+}
+
+.navigation__link-text {
+  margin-top: 7px;
+  display: block;
+  padding: 15px 25px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
```
Comment (mi-chal) [inline src/shared/components/Navigation/styles.scss:24]: Colors should be imported from `colors.scss`

File: src/shared/components/Navigation/styles.scss [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+  font-size: 1rem;
+  border-bottom: 4px solid white;
+  width: 140px;
+  text-align: center;
+}
+
+.active {
+  color: #ED7533;
+  border-bottom-color: #ED7533;
```
Comment (mi-chal) [inline src/shared/components/Navigation/styles.scss]: Colors should be imported from `colors.scss`

File: src/shared/components/Navigation/styles.scss [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+  text-transform: uppercase;
+  color: black;
+  font-size: 1rem;
+  border-bottom: 4px solid white;
+  width: 140px;
+  text-align: center;
+}
+
+.active {
+  color: #ED7533;
```
Comment (mi-chal) [inline src/shared/components/Navigation/styles.scss]: Colors should be imported from `colors.scss`

File: src/shared/components/Navigation/styles.scss [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+
+.navigation__link-text {
+  margin-top: 7px;
+  display: block;
+  padding: 15px 25px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+  font-size: 1rem;
+  border-bottom: 4px solid white;
```
Comment (mi-chal) [inline src/shared/components/Navigation/styles.scss]: Colors should be imported from `colors.scss`

File: src/App.jsx (hunk 4 above)
Comment (dwantula) [reply src/App.jsx]: Fixed

File: src/index.scss (hunk 1 above)
Comment (dwantula) [reply src/index.scss]: Done

File: src/App.jsx (hunk 5 above)
Comment (dwantula) [reply src/App.jsx]: Fixed

File: src/App.jsx (hunk 3 above)
Comment (dwantula) [reply src/App.jsx]: Done

File: src/index.jsx (hunk 2 above)
Comment (dwantula) [reply src/index.jsx]: Fixed

File: src/screens/AbcCams/AbcCams/styles.scss (hunk 6 above)
Comment (dwantula) [reply src/screens/AbcCams/AbcCams/styles.scss]: Done

File: src/screens/AbcCams/AbcCams/styles.scss (hunk 7 above)
Comment (dwantula) [reply src/screens/AbcCams/AbcCams/styles.scss]: Done

File: src/screens/AbcCams/AbcCams/styles.scss (hunk 8 above)
Comment (dwantula) [reply src/screens/AbcCams/AbcCams/styles.scss]: Done

File: src/shared/components/Navigation/styles.scss (hunk 9 above)
Comment (dwantula) [reply src/shared/components/Navigation/styles.scss:24]: Done

File: src/shared/components/Navigation/styles.scss (hunk 12 above)
Comment (dwantula) [reply src/shared/components/Navigation/styles.scss]: Done

File: src/shared/components/Navigation/styles.scss (hunk 11 above)
Comment (dwantula) [reply src/shared/components/Navigation/styles.scss]: Done

File: src/shared/components/Navigation/styles.scss (hunk 10 above)
Comment (dwantula) [reply src/shared/components/Navigation/styles.scss]: Done

**Pull request merged**